packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -409,8 +418,17 @@ class TxForm extends Component {
       };
     } else if (balance && balance.lt(amountBn)) {
       return { amount: `You don't have enough ${token.symbol} balance` };
-    } else if (!values.to || !isAddress(values.to)) {
+    } else if (
+      (!values.to || !isAddress(values.to)) &&
+      currentChainIdBN.valueOf() !== '61'
+    ) {
       return { to: 'Please enter a valid Ethereum address' };
+      // FIXME - modify to check if it is a valid ETC address
```

Comment:
we can't deduct to what network (eth/etc) the address corresponds based solely on the address. so logic for address validation for eth/etc is the same. a valid eth address is a valid etc address